thorlcr/slave/thslavemain.cpp

```diff
@@ -583,7 +576,6 @@ int main( int argc, const char *argv[]  )
 #ifndef _CONTAINERIZED
     stopPerformanceMonitor();
 #endif
-    ClearTempDirs();
```

Comment:
So we are doing this in endJob now ?
What if anything in that try throws ?  Should we ClearTempDirs() outside the catch ?

Reply:
@mckellyln - ClearTempDir does handle exceptions (and logs only) won't throw, so it's fine where it is.